packages/babel-parser/src/parser/error-message.js

```diff
@@ -134,18 +134,22 @@ export const ErrorMessages = makeErrorTemplates(
     ParamDupe: "Argument name clash.",
     PatternHasAccessor: "Object pattern can't contain getter or setter.",
     PatternHasMethod: "Object pattern can't contain methods.",
-    PipelineBodyNoArrow:
+    PipeBodyIsTight:
+      "Unexpected %0 after pipeline body; any %0 expression acting as Hack-style pipe body must be parenthesized due to its loose operator precedence.",
+    // The `PipeSmartMix` error messages are retained for backwards compatibility
+    // with the deprecated smart-mix pipe operator proposal plugin.
+    PipeSmartMixBodyCannotBeArrow:
       'Unexpected arrow "=>" after pipeline body; arrow function in pipeline body must be parenthesized.',
-    PipelineBodySequenceExpression:
-      "Pipeline body may not be a comma-separated sequence expression.",
-    PipelineHeadSequenceExpression:
-      "Pipeline head should not be a comma-separated sequence expression.",
-    PipelineTopicUnused:
-      "Pipeline is in topic style but does not use topic reference.",
-    PrimaryTopicNotAllowed:
+    PipeSmartMixPrimaryTopicNotAllowed:
       "Topic reference was used in a lexical context without topic binding.",
-    PrimaryTopicRequiresSmartPipeline:
-      "Primary Topic Reference found but pipelineOperator not passed 'smart' for 'proposal' option.",
+    PipeSmartMixTopicUnused:
```

Comment:
```suggestion
    PipelineTopicUnused:
```

Reply:
This has been addressed in 86c18d3dba0972d2447eff2bd23fe3422d95efcb.